cupy/testing/helper.py

```diff
@@ -408,7 +408,7 @@ def numpy_cupy_equal(name='xp', sp_name=None):
              ``scipy.sparse`` or ``cupy.sparse`` module. If ``None``, no
              argument is given for the modules.
 
-    Decorated test fixture is required throw same errors
+    Decorated test fixture is required return the same results
```

Comment:
How about "*to* return"?

Reply:
Sorry and thank you, fixed as suggested.